src/github/index.ts

```diff
@@ -5,14 +5,24 @@ import pullRequest from './pull-request';
 import push from './push';
 import { createBranch, deleteBranch } from './branch';
 import webhookTimeout from '../middleware/webhook-timeout';
+import bunyan from 'bunyan';
+import statsd from '../config/statsd';
 
 export default (robot) => {
+  const logger = bunyan.createLogger({ name: 'github' });
+
   // TODO: Need ability to remove these listeners, especially for testing...
   robot.on('*', (context) => {
-    context.log(
-      { event: context.name, action: context.payload.action },
-      'Event received',
-    );
+    const { name, payload } = context;
+    logger.info({ event: name, action: payload.action }, 'Event received');
+
+    const tags = [
+      `name: webhooks`,
+      `event: ${name}`,
+      `action: ${payload.action}`,
+    ];
+
+    statsd.increment('jira_configuration_error', tags);
```

Comment:
I wonder would `jira_configuration_error` be a metric name or there will be some kind of prefix?

Our metrics should follow some kind of naming convention

For exmaple in DSS we call them `counter.dss.commit.processor.webhook.truncated`

I think here we can also  come up with somethning like:

`github.for.jira.app.server.error.jira.configuration`

`github.for.jira.app.server.http.request.handling.duration`

Reply:
![Screen Shot 2021-06-28 at 12 52 49 pm](https://user-images.githubusercontent.com/37155488/123572554-c370f100-d80f-11eb-8d3b-4abe6ee6b174.png)

All metrics are prefixed with `github-for-jira.` as defined in the global tags. I'll update the individual metrics themselves, however, to follow a better convention.